docs/sql/statements/create-user.rst

```diff
@@ -79,3 +79,14 @@ The following ``user_parameter`` are supported to define a new user account:
   ::
 
      CREATE USER john password 'foo'
+
+:jwt:
+  JWT properties map ('iss' and 'username') entered as string literal. e.g.::
+
+     CREATE USER john WITH (jwt = {"iss" = 'https://login.company.org/keys', "username" = 'test@company.org'})
+
+  `iss`_ is a JWK endpoint, containing public keys.
+  ``username`` is a user name in a third party app. Not neccearily have to match CrateDb user.
+  Combination of ``iss`` and ``username`` must be unique.
+
+.. _iss: https://www.rfc-editor.org/rfc/rfc7519#section-4.1.1
```

Comment:
Once signature verification with docs is merged, I will also add here "see also" (ref to jwt_auth docs) link.